.github/workflows/release.yaml

```diff
@@ -34,6 +83,9 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
         run: yarn publish
-
-      - name: Push version tag
-        run: yarn release:tag
+      
```

Comment:
Looks like it's missing the `yarn docs:deploy` step?

Reply:
Added